### PR TITLE
feat(server): publish multiple items at once

### PR DESCRIPTION
**Pull request by nourbalaha**
# Overview
- Publish multiple items instead of one item
- Rename itemId to itemIds in `UnpublishItemInput`

### CODE REVIEW

File: server/internal/usecase/interactor/item_test.go [hunk 1]
```diff
@@ -1073,4 +1073,22 @@ func TestWorkFlow(t *testing.T) {
 	status, err = itemUC.ItemStatus(ctx, id.ItemIDList{i.ID()}, op)
 	assert.NoError(t, err)
 	assert.Equal(t, map[id.ItemID]item.Status{i.ID(): item.StatusDraft}, status)
```
Comment (rot1024) [inline server/internal/usecase/interactor/item_test.go:1075]:
```suggestion
	assert.Equal(t, map[id.ItemID]item.Status{i.ID(): item.StatusDraft}, status)

```

File: server/internal/usecase/interactor/item.go [hunk 2]
```diff
@@ -419,21 +419,30 @@ func (i Item) Unpublish(ctx context.Context, itemIDs id.ItemIDList, operator *us
...
+		// check all items were found
+		if len(items) != len(itemIDs) {
+			return nil, interfaces.ErrItemMissing
+		}
+
+		// check all items on the same models
+		s := lo.CountBy(items, func(itm item.Versioned) bool {
+			return itm.Value().Model() == items[0].Value().Model()
+		})
+		if s != len(items) {
```
Comment (rot1024) [inline server/internal/usecase/interactor/item.go:441]:
```suggestion
		if lo.EveryBy(items, func(itm item.Versioned) bool {
			return itm.Value().Model() == items[0].Value().Model()
		}) {
```

**Review by rot1024 — APPROVED** [review]
Front-end should be updated also

File: server/internal/usecase/interactor/item.go [hunk 3]
```diff
@@ -419,21 +419,30 @@ func (i Item) Unpublish(ctx context.Context, itemIDs id.ItemIDList, operator *us
...
+	return Run1(ctx, operator, i.repos, Usecase().Transaction(), func(ctx context.Context) (item.VersionedList, error) {
+		items, err := i.repos.Item.FindByIDs(ctx, itemIDs, nil)
 		if err != nil {
 			return nil, err
 		}
 
-		if itm.Refs().Has(version.Public) {
-			return nil, interfaces.ErrAlreadyPublished
+		// check all items were found
+		if len(items) != len(itemIDs) {
```
[the remark this answers is not in the document]
Comment (nourbalaha) [reply server/internal/usecase/interactor/item.go:433]: Done!

File: server/internal/usecase/interactor/item.go (hunk 2 above)
Comment (nourbalaha) [reply server/internal/usecase/interactor/item.go:441]: Done!

File: server/internal/usecase/interactor/item_test.go (hunk 1 above)
Comment (nourbalaha) [reply server/internal/usecase/interactor/item_test.go:1075]: Done!

**Pull request merged**